### PR TITLE
Winter 2022

### PR DESCRIPTION
**Pull request by LinguList**
# Pull request checklist

- [x] add new concept list
- [ ] add new metadata
- [ ] add new Concepticon concept sets
  * [ ] checked whether the new concept(s) can be applied to existing lists with
    `concepticon notlinked --gloss "NEW_GLOSS"`
- [ ] add new Concepticon concept relations
- [ ] refine existing Concepticon concept set mappings
- [ ] refine Concepticon glosses
- [ ] refine Concepticon concept relations
- [ ] refine Concepticon concept definitions
- [ ] retire data

## Additional information

@xrotwang, I have now not changed names, but just use json to indicate the format. Since it is a link between concepts that may be used to infer links between concept sets, I left the "id" in json, since this seems to be most consistent (we could use "concept_id" as well). I also added a folder that shows how I download the data and edit them, since I consider this step as important for similar lists, where we want more transparency in this stage. 

What I'd like to add to pyconcepticon is a test or an extension of the `check` subcommand that will check network properties of these structures. Should I make a PR here, or should we discuss it first in an issue? My idea would be to reserve the column name `TARGET_CONCEPTS` to json that has an id that links to other concepts in the concept list. I'd also consider adding a command that converts the concept list to an edge list, as it could be used in Cytoscape and other software, where the JSON is then interpreted. We could use `LINKED_CONCEPTS` for all cases where the relation is not directed.

If we agree on the format above, I'd also modify `Urban-2011-160` accordingly. 

@AnnikaTjuka and @katjabocklage, if you could review my mappings (another time, sorry), this would be helpful.

### CODE REVIEW

**Review by AnnikaTjuka — COMMENTED** [review]
Thanks for preparing the list, @LinguList! I added a few suggestions.

File: concepticondata/conceptlists/Winter-2022-99.tsv [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+Winter-2022-99-73	73	pen	794	PEN	"[{""name"": ""feather"", ""id"": ""Winter-2022-99-74"", ""polysemy"": 3, ""overt_marking"": 0}]"	"[{""name"": ""feather"", ""id"": ""Winter-2022-99-74"", ""polysemy"": 3, ""overt_marking"": 3}]"
+Winter-2022-99-74	74	feather	1201	FEATHER	"[{""name"": ""pen"", ""id"": ""Winter-2022-99-73"", ""polysemy"": 3, ""overt_marking"": 3}, {""name"": ""beard"", ""id"": ""Winter-2022-99-75"", ""polysemy"": 2, ""overt_marking"": 10}]"	"[{""name"": ""pen"", ""id"": ""Winter-2022-99-73"", ""polysemy"": 3, ""overt_marking"": 0}, {""name"": ""beard"", ""id"": ""Winter-2022-99-75"", ""polysemy"": 2, ""overt_marking"": 0}]"
+Winter-2022-99-75	75	beard	678	BEARD	"[{""name"": ""feather"", ""id"": ""Winter-2022-99-74"", ""polysemy"": 2, ""overt_marking"": 0}]"	"[{""name"": ""feather"", ""id"": ""Winter-2022-99-74"", ""polysemy"": 2, ""overt_marking"": 10}]"
+Winter-2022-99-76	76	meteoroid			"[{""name"": ""star"", ""id"": ""Winter-2022-99-77"", ""polysemy"": 3, ""overt_marking"": 0}]"	"[{""name"": ""star"", ""id"": ""Winter-2022-99-77"", ""polysemy"": 3, ""overt_marking"": 28}]"
```
[the remark this answers is not in the document]
Comment (LinguList) [reply concepticondata/conceptlists/Winter-2022-99.tsv:77]: I had it mapped automatically even, the I probably deleted it.

File: concepticondata/conceptlists/Winter-2022-99.tsv [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+Winter-2022-99-81	81	rain	658	RAIN (PRECIPITATION)	"[{""name"": ""sky"", ""id"": ""Winter-2022-99-82"", ""polysemy"": 3, ""overt_marking"": 0}, {""name"": ""cloud"", ""id"": ""Winter-2022-99-1"", ""polysemy"": 4, ""overt_marking"": 2}]"	"[{""name"": ""sky"", ""id"": ""Winter-2022-99-82"", ""polysemy"": 3, ""overt_marking"": 2}, {""name"": ""cloud"", ""id"": ""Winter-2022-99-1"", ""polysemy"": 4, ""overt_marking"": 2}]"
+Winter-2022-99-82	82	sky	1732	SKY	"[{""name"": ""rain"", ""id"": ""Winter-2022-99-81"", ""polysemy"": 3, ""overt_marking"": 2}, {""name"": ""cloud"", ""id"": ""Winter-2022-99-1"", ""polysemy"": 8, ""overt_marking"": 11}]"	"[{""name"": ""rain"", ""id"": ""Winter-2022-99-81"", ""polysemy"": 3, ""overt_marking"": 0}, {""name"": ""cloud"", ""id"": ""Winter-2022-99-1"", ""polysemy"": 8, ""overt_marking"": 2}]"
+Winter-2022-99-83	83	spring	1537	SPRINGTIME	"[{""name"": ""river/stream"", ""id"": ""Winter-2022-99-28"", ""polysemy"": 12, ""overt_marking"": 0}]"	"[{""name"": ""river/stream"", ""id"": ""Winter-2022-99-28"", ""polysemy"": 12, ""overt_marking"": 2}]"
```
[the remark this answers is not in the document]
Comment (LinguList) [reply concepticondata/conceptlists/Winter-2022-99.tsv:84]: Yes!

**Comment by LinguList** [comment]
So the list is called winter-2022-98 now ;-)

**Comment by LinguList** [comment]
I added Urban's modified list now as well. We'd need checks as a second point now.

File: concepticondata/conceptlists/Urban-2011-160.tsv [hunk 3]
```diff
@@ -1,161 +1,161 @@
...
+Urban-2011-160-130	130	nostrils	1178	NOSTRIL	III	Body Parts and Body Fluids	[]
+Urban-2011-160-131	131	pupil	1658	PUPIL	III	Body Parts and Body Fluids	[]
+Urban-2011-160-132	132	pus	1808	PUS	III	Body Parts and Body Fluids	[]
+Urban-2011-160-133	133	rib	801	RIB	III	Body Parts and Body Fluids	[]
+Urban-2011-160-134	134	saliva/spittle	1503	SALIVA	III	Body Parts and Body Fluids	"[{""id"": ""Urban-2011-160-24"", ""name"": ""foam"", ""polysemy"": 8, ""overt_marking"": 4, ""shift_id"": 35}]"
+Urban-2011-160-135	135	scar	497	SCAR	III	Body Parts and Body Fluids	[]
+Urban-2011-160-136	136	skin	763	SKIN	III	Body Parts and Body Fluids	"[{""id"": ""Urban-2011-160-3"", ""name"": ""bark"", ""polysemy"": 45, ""overt_marking"": 26, ""shift_id"": 33}]"
+Urban-2011-160-137	137	snot	208	NASAL MUCUS (SNOT)	III	Body Parts and Body Fluids	[]
+Urban-2011-160-138	138	semen	2283	SEMEN	III	Body Parts and Body Fluids	[]
+Urban-2011-160-139	139	sweat	2458	PERSPIRE OR SWEAT	III	Body Parts and Body Fluids	[]
```
Comment (AnnikaTjuka) [inline concepticondata/conceptlists/Urban-2011-160.tsv:140]: I'd map this to 125 SWEAT (SUBSTANCE).

**Review by AnnikaTjuka — COMMENTED** [review]
There is just one mapping in Urban's list where I suggest an alternative concept. Otherwise, everything looks good from my side.

**Comment by LinguList** [comment]
I wait for @xrotwang to check here on my other question before merging then.

**Review by AnnikaTjuka — APPROVED** [review]

**Comment by LinguList** [comment]
I also changed the text in conceptlists.tsv for Urban now. If we use this as a regular format for concept lists, we will have some work to retro-standardize this ;-)

**Comment by LinguList** [comment]
I also added a check-command now, under review in pyconcepticon, with this command, you can type:

```
concepticon check conceptlist
```

and it yields a basic sanity check of graph data that I consider useful. Next step would be a command that creates an edge table so one can load data into cytoscape, or a command to visualize the network quickly to make sure it makes sense.

File: concepticondata/conceptlists/Urban-2011-160.tsv [hunk 4]
```diff
@@ -1,161 +1,161 @@
...
+Urban-2011-160-128	128	neck	1333	NECK	III	Body Parts and Body Fluids	[]
+Urban-2011-160-129	129	nipple	796	NIPPLE	III	Body Parts and Body Fluids	[]
+Urban-2011-160-130	130	nostrils	1178	NOSTRIL	III	Body Parts and Body Fluids	[]
+Urban-2011-160-131	131	pupil	1658	PUPIL	III	Body Parts and Body Fluids	[]
+Urban-2011-160-132	132	pus	1808	PUS	III	Body Parts and Body Fluids	[]
+Urban-2011-160-133	133	rib	801	RIB	III	Body Parts and Body Fluids	[]
+Urban-2011-160-134	134	saliva/spittle	1503	SALIVA	III	Body Parts and Body Fluids	"[{""id"": ""Urban-2011-160-24"", ""name"": ""foam"", ""polysemy"": 8, ""overt_marking"": 4, ""shift_id"": 35}]"
+Urban-2011-160-135	135	scar	497	SCAR	III	Body Parts and Body Fluids	[]
+Urban-2011-160-136	136	skin	763	SKIN	III	Body Parts and Body Fluids	"[{""id"": ""Urban-2011-160-3"", ""name"": ""bark"", ""polysemy"": 45, ""overt_marking"": 26, ""shift_id"": 33}]"
+Urban-2011-160-137	137	snot	208	NASAL MUCUS (SNOT)	III	Body Parts and Body Fluids	[]
```
Comment (katjabocklage) [inline concepticondata/conceptlists/Urban-2011-160.tsv:138]: Both mucus (126) and snot (137) are mapped to NASAL MUCUS (SNOT). Doesn't this affect the concept count?

**Comment by LinguList** [comment]
Is there "mucus" in Concepticon without "nasal"? I would have to check. 
This looks like a mapping error.

**Comment by katjabocklage** [comment]
> Is there "mucus" in Concepticon without "nasal"? I would have to check. This looks like a mapping error.

An alternative I could find is PHLEGM, it doesn't include "nasal" but still concerns the respiratory tract.

**Comment by LinguList** [comment]
Then we should map to PHLEGM for now to keep both distinct. But you could open an issue, @katjabocklage, asking the other members, if they would prefer to have a specific concept "mucus" that does not refer to the mucus from the nose, but would be more general (if this is the intention in Urban's list). What do you think, @AnnikaTjuka ?

**Comment by LinguList** [comment]
@AnnikaTjuka, I have now also added the graph from the colexification paper. I should also add the old CLICS graphs. But that most not happen in the next week, rather something we do step by step from next year on.

You can now with the new branch with a pr in pyconcepticon also explort teh concept list to an edge list:

```
concepticon graph List-2023-1308.tsv --format=tsv  --threshold=20 --threshold-property=AffixFams --graph-column=TARGET_CONCEPTS
```
This filters affix-colexifications by families >= 20.

**Comment by LinguList** [comment]
The resulting edges.tsv can be read directly (specifying the columns upon import) in cytoscape.

**Comment by AnnikaTjuka** [comment]
@LinguList Thanks for adding the new list! I think we can add the CLICS lists with a new PR so that this one can be merged.

**Comment by LinguList** [comment]
We must in any case review the format for representing concept graphs. Plus, the new proposal of adding a folder with the conversion script itself already to concepticon is also something worth discussion, so my suggestion would be that we close this PR / merge it, once we reached a conclusion here. The List-list also shows the consequences, as the list is restricted by a threshold and goes in the MB in terms of size. So we must decide if we want this complexity for concepticon, or if we rather prefer to store the networks in independent CLDF datasets.

**Comment by LinguList** [comment]
Before we get more data, I suggest to merge the data now. We have four examples for JSON-networks in our curation format for concepticon. The publication format will be CLDF and would then make use of those.

**Pull request merged**